docs/user-guide/pod-preset/index.md

```diff
@@ -0,0 +1,540 @@
+---
+assignees:
+- jessfraz
+title: Pod Preset
+---
+
+Objects of type `podpreset` allow for injecting certain information into pods
+at creation time such as secrets, volumes, volume mounts and environment
+variables.
+
+See [PodPreset proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/pod-preset.md) for more information.
+
+* TOC
+{:toc}
+
+## What is a Pod Preset?
+
+A _Pod Preset_ is an API resource that allows additional runtime requirements
+for Pods to be injected at creation time based off label selectors.
+
+Using a Pod Preset allows pod template authors to not have to explicitly set
+information for every pod. This way authors of pod templates consuming a
+specific service do not need to know all the details about that service.
+
+## Admission
+
+_Admission control_ with `PodPreset` introspects all incoming pod creation
```

Comment:
This could be clearer.

"_Admission control_ is how Kubernetes applies Pod Presets to incoming pod creation requests. When a pod creation request occurs, the system does the following:"